documentation/bin/createDocu.groovy

```diff
@@ -523,9 +533,13 @@ def handleStep(stepName, prepareDefaultValuesStep, gse) {
 
     System.err << "[INFO] Handling step '${stepName}'.\n"
 
+    Map defaultValueParams = [:]
```

Comment:
[naming] the name here is IMO very confusing. From that name I would guess that the default values are provided with the map directly. But basically this map is the well known parameters map which is handed over to each step. And one entry in that map (... a parameter: `customDefaults` holds a path to a file which provides additional defaults). In order to make life easier for the developers coming behind us I would like to suggest to use a more speaking name here.

Reply:
renamed to `prepareDefaultValuesStepParams`